backend/api/views.py

```diff
@@ -130,99 +145,136 @@ def put(self, request, cashflow_id):
-            },
-            {
-                'Endpoint': '/users/login',
-                'method': 'POST',
-                self.tokenNeeded: 'false',
-                'body': {'email': '', 'password': ''},
-                'description': 'To log in, the email and password specified during registration must be sent along. The method returns the Refresh and Access Token.'
-            },
-            {
-                'Endpoint': '/users/authenticate',
-                'method': 'POST',
-                self.tokenNeeded: 'Refresh',
-                'body': {'refresh': ''},
-                'description': 'To authenticate, a valid refresh token needs to be entered. The method returns an Access Token.'
-            },
-            {
-                'Endpoint': '/users/delete',
-                'method': 'DELETE',
-                self.tokenNeeded: 'Access',
-                'body': None,
-                'description': 'Deletes user that corresponds to the Access Token send in the header'
-            },
-            {
-                'Endpoint': '/users/get',
-                'method': 'GET',
-                self.tokenNeeded: 'Access',
-                'body': None,
-                'description': 'Returns user that corresponds to the Access Token send in the header'
-            },
-            {
-                'Endpoint': '/users/update',
-                'method': 'PUT',
-                self.tokenNeeded: 'Access',
-                'body': {'email': '', 'newPassword': '', 'oldPassword': ''},
-                'description': 'Not implemented yet. Updates current user with data sent in put request'
-            },
-        ]
-        return Response(routes)
+
+    def post(self, request):
+        try:
+            account = Account.objects.get(id=request.data['account'])
+            private = Private.objects.create(account=account, first_name=request.data['private']['first_name'],
+                                             last_name=request.data['private']['last_name'])
+        except Exception as e:
+            # Fixme differentiate between other exception types
+            print(e.__cause__)
+            return JsonResponse({'success': False}, status=status.HTTP_400_BAD_REQUEST)
+
+        return JsonResponse({'success': True, 'private': private.id}, status=status.HTTP_201_CREATED)
+
+    def get(self, request, private_id):
+        try:
+            private = Private.objects.get(id=private_id)
+        except Exception as e:
+            # Fixme differentiate between other exception types
+            print(e.__cause__)
+            return JsonResponse({'success': False}, status=status.HTTP_404_NOT_FOUND)
+
+        serializer = PrivateSerializer(private, many=False)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ItemView(APIView):
+    # permission_classes = (IsAuthenticated,)
+
+    def post(self, request, cashflow_id):
+        try:
```

Comment:
Please add a FIXME that authorization should be checked (i.e. whether the user may access this cashflow by managing the corresponding account).